datalad/interface/run.py

```diff
@@ -302,10 +303,11 @@ def run_command(cmd, dataset=None, inputs=None, outputs=None, expand=None,
         'cmd': cmd,
         'exit': cmd_exitcode if cmd_exitcode is not None else 0,
         'chain': rerun_info["chain"] if rerun_info else [],
-        'inputs': inputs,
+        'inputs': [relpath(p, start=ds.path) if isabs(p) else p for p in inputs],
```

Comment:
Shouldn't start be `pwd` rather than `ds.path`?

Reply:
Is the `rerun` `get` call resolved via `pwd`, or via `ds.path` ( I guess the latter, if you are stating this)?

Reply:
> Is the rerun get call resolved via pwd, or via ds.path

Eh, via ds.path, which means there's a bug because globbing is done relative to `pwd`. If you glob within a subdirectory, it will be detected as an input but the the get call will say it's not there.  So, your code is in line with get/unlock/remove, and my glob code is buggy.

Reply:
Thinking about this more, I find storing the inputs relative to `pwd` to be more intuitive, especially once we add the inputs/outputs placeholders to the command.  I can fix the subdir glob bug in a separate PR.